scyther_on_mac.md

```diff
@@ -0,0 +1,51 @@
+1. Open the Terminal
+2. Install Homebrew, the  macOS package manager, by running the following command:
+   
+    “/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)””
+
+3. Add it to the PATH by using the following commands,
+
+    “echo >> /Users/kashishgaur/.zprofile”,
+
+    “echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/machinename/.zprofile”,
+
+    “eval "$(/opt/homebrew/bin/brew shellenv)””
+
+    Replace “manchinename” with your respective machine name
+    You can also find these steps when the installation of Homebrew is completed, under the “Next Steps”
+
+4. To check if Homebrew is working as intended or not, we can run the following command,
+
+    “brew help”, which should display the example usage of brew.
+
+5. Move to Desktop and clone scyther,
+
+   “cd Desktop”
+
+    “git clone https://github.com/cascremers/scyther.git”
+
+6. Move to the scyther folder, “cd scyther”
+
+7. Run the following command to check the tag,
+    “git tag”, and make sure that there exists a v1.2 
+
+8. Go the ‘gui’ folder, and run the following command to install wxPython,
+    “python3 scyther-gui.py”, this will automatically install wxPython on the machine.
+
+9. To install graphviz, install it by,
+
+    “brew install graphviz”
+
+10. Move to the src directory, and run the following command to build scyther on the machine,
+
+    “cmake -G "Unix Makefiles" ”
```

Comment:
Doesn't just "cmake" (without arguments) work already?

Reply:
(I'm actually not even sure if cmake is needed at all here, just `make` seems to work fine.)

Reply:
Yes, simply "cmake" works fine too.
about the second query, it's usually recommended to run 'cmake' before 'make', especially if we use CMake to generate the build system, which is what we are doing. we want to build the system before executing it.
Just to prevent any unnecessary issues, which users might face (if the build environment is not generated correctly), I've chosen to run the 'cmake' command before the 'make' command